api_inference_community/validation.py

```diff
@@ -244,18 +249,56 @@ def check_inputs(inputs, tag):
 }
 
 
+WHITELISTED_MIME_TYPES = {
+    "audio/flac": "flac",
+    "audio/mpeg": "mp3",
+    "audio/wav": "wav",
+    "audio/ogg": "ogg",
+    "audio/mp4": "m4a",
+    "audio/aac": "aac",
+    "audio/webm": "webm",
+    "image/jpeg": "jpg",
+    "image/png": "png",
+    "image/bmp": "bmp",
+    "image/tiff": "tiff",
+    "image/webp": "webp",
```

Comment:
Can we split this map into 2 ? 1 for audio 1 for images ? Currently a user could ask tiff on mimetypes making the thing crash on ffmpeg.

Splitting into 2 could also help remove the `image/` /  `audio/` prefix (which you are readding later anyway, so we could drop from dict to set leading to easier lookups (instead of doing seq scans in dict.values()).

The perf doesn't matter here, but the code should end up being cleaner

Reply:
>  Currently a user could ask tiff on mimetypes making the thing crash on ffmpeg.

oof, good catch, I also wanna write a unit test for this.

Reply:
@Narsil I split up the map as suggested